exporter/debugexporter/factory.go

```diff
@@ -53,7 +53,7 @@ func createDefaultConfig() component.Config {
 func createTraces(ctx context.Context, set exporter.Settings, config component.Config) (exporter.Traces, error) {
 	cfg := config.(*Config)
 	exporterLogger := createLogger(cfg, set.Logger)
-	debug := newDebugExporter(exporterLogger, cfg.Verbosity)
+	debug := newDebugExporter(exporterLogger, cfg.Verbosity, cfg.Output)
```

Comment:
Perhaps just pass the whole config as a single variable?
```suggestion
	debug := newDebugExporter(exporterLogger, cfg)
```